File: php/operations/HelloWorldOperation.php
<?php

require_once 'Operation.php';

class HelloWorldOperation extends Operation {

    private $output;

    public function __construct($db, $output)
    {
        parent::__construct("Hello World", $db);
        $this->output = $output;
    }

    function execute() {
        fwrite($this->output, $this->db->hello_world() . "\n"); 
    }

    function revert() {
        fwrite($this->output, $this->db->hello_world() . " reverted!\n"); 
    }
}

File: dotnet/as-6-litedata/operations/HelloWorldOperation.cs
using System;
using as_6_litedata.models;

namespace as_6_litedata.operations
{
    public class HelloWorldOperation : Operation
    {

        public HelloWorldOperation(Database db) : base("Hello World", db)
        {
        
        }

        public override void Execute()
        {
            System.IO.File.AppendAllText("output.txt", db.HelloWorld() + "\n");
            
        }

        public override void Revert()
        {
            System.IO.File.AppendAllText("output.txt", db.HelloWorld() + " reverted!\n");

        }
    }

}
